src/randomize.jl

```diff
@@ -47,13 +55,18 @@ Swap random tuples within `tuples`.
 Based on formula from Handorf et al. 2008.
 """
 function mix_it_up!(
-    tuples::Vector{<:Tuple{String,Real}},
+    tuples::Vector{<:Tuple{String,Any}},
     beta::Real,
     n_swaps::Int,
     rng::AbstractRNG=Random.default_rng()) #where {R <: Real}
 
     for _ in 1:n_swaps
-        i,j = rand(1:length(tuples),2) ## 2 random indices 
+        i,j = rand(1:length(tuples),2) ## two random indices
+        if i > j
+            i,j = j,i
```

Comment:
Doesn't this make it so that the `elseif` in the `getflipprobability` will never get triggered? Is that good or bad?

Reply:
Thank you for checking it. This is to reduce the number of exchanges of compounds.
If the `i>j` cases are allowed, `diff = ti[2] - tj[2]` tends to be negative (Positions of compound i and compound j are like [c1, ... , cj, ... , ci, ... , cn]). As a result, compounds are much more likely to be flipped.
The attached figure shows the positions of compounds after random perturbation and their mass (a) when the `i>j` cases are allowed and (b) when not allowed.
I thought it's good to keep the trend of the initial order of compounds as figure b, so index j is to be larger than i in the modified code. `elseif` in the `getflipprobability` happens when compound i is heavier than compound j.
![order_of_compound_after_perturbation](https://github.com/hbsmith/BioXP/assets/121482943/6165e02b-a088-407f-9cfc-49dbc49342d9)

Reply:
got it! thanks for explaining/reminding me. everything looks good then